docs/guides/code/conditional_actor_charge.ts

```diff
@@ -0,0 +1,18 @@
+import { Actor } from 'apify';
+
+await Actor.init();
+
+let chargedItems = 0;
```

Comment:
```suggestion
// Check the dataset because there might already be items if the run migrated or was restarted
const defaultDataset = Actor.openDataset();
let chargedItems = (await defaultDataset.getInfo()!).itemCount;
```